src/main/java/core/basesyntax/strategy/ReturnOperationHandler.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.serviceimpl.FruitTransactionValidatorImpl;
+import java.util.Optional;
+
+public class ReturnOperationHandler implements OperationHandler {
+    private FruitTransactionValidatorImpl validator;
+
+    @Override
+    public void apply(FruitTransaction fruitTransaction) {
+        validator.validateOperation(fruitTransaction);
```

Comment:
There is a redundant call to `validateOperation` here. Since the `ReturnOperationHandler` is only responsible for the 'RETURN' operation, there's no need to validate the operation type again.